cvat/apps/dataset_manager/annotation.py

```diff
@@ -697,11 +693,7 @@ def polyshape_interpolation(shape0, shape1):
             distance = shape1["frame"] - shape0["frame"]
             for frame in range(shape0["frame"] + 1, shape1["frame"]):
                 offset = (frame - shape0["frame"]) / distance
-                points = None
-                if shape1["outside"]:
-                    points = np.asarray(shape0["points"])
-                else:
-                    points = interpolate_position(shape0, shape1, offset)
+                points = interpolate_position(shape0, shape1, offset)
```

Comment:
@kirill-sizov , I believe we don't need to interpolate if the first shape has outside=True.

Reply:
@nmanovic, yes, you're right, but this case is handled earlier before calling the interpolate function, here:
https://github.com/openvinotoolkit/cvat/blob/e76b0ea5fc31509b0462694ddfd6bbc982492168/cvat/apps/dataset_manager/annotation.py#L733-L734